src/utils.ts

```diff
@@ -1,9 +1,17 @@
+import yaml from 'js-yaml';
+import fs from 'fs';
+import path from 'path';
+import Template from '@/interfaces/Template';
+
 export default class Utils {
-    public static getPlatformConfig(platformId: string): any {
-        return require(`../platforms/${platformId}/config.json`);
-    }
+  public static getPlatformConfig(platformId: string): any {
+    return require(`../platforms/${platformId}/config.json`);
+  }
 
-    public static getTemplate(platformId: string, templateId: string, type: string): any {
-        return require(`../platforms/${platformId}/templates/${templateId}/${type}.json`);
-    }
+  public static loadTemplate(platformId: string, templateId: string, type: string): Template {
+    const templatePath = path.join(__dirname, '..', 'platforms', platformId, 'templates', templateId, `${type}.mustache`);
```

Comment:
`WARNING: --/utils.ts[12, 1]: Exceeds maximum line length of 120`
Can you fix this?